### PR TITLE
feature(viewer): read `csi` from viewer params

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
we'd want to merge this **after** the viewer already has the new param since i set the default to be "off" (which we can reverse of course)

File: examples/viewer.html [hunk 1]
```diff
@@ -194,7 +194,8 @@
         paddingTop: this.header./*OK*/offsetHeight,
         visibilityState: this.visibilityState_,
         prerenderSize: this.prerenderSize_,
-        viewerorigin: parseUrl(window.location.href).origin
+        viewerorigin: parseUrl(window.location.href).origin,
+        csi: 0
```
Comment (cramforce) [inline examples/viewer.html]: lets call this property `performanceTracking`. At least on our side.

**Comment by cramforce** [comment]
LGTM with one style comment

File: examples/viewer.html (hunk 1 above)
Comment (erwinmombay) [inline examples/viewer.html]: done.

added some additional code in `performance.js` to ignore all `ticks` and `flush` calls when performanceTracking is off.

**Pull request merged**